### PR TITLE
Adapt S3 configuration to ADR016

### PR DESCRIPTION
**Pull request by siegfriedweber**
## Description

S3 configuration adapted to ADR016

Trino PR: https://github.com/stackabletech/trino-operator/pull/200

Currently requires a certain secret operator pr:
`helm install secret-operator stackable-test/secret-operator --version=0.3.0-pr125`

## Review Checklist
- [ ] Code contains useful comments
- [ ] Changelog updated (or not applicable)

### CODE REVIEW

**Review by maltesander — COMMENTED** [review]
I think you can just leave the s3 part out? The question is do we wanna have integration tests for that s3 stuff?

**Comment by siegfriedweber** [comment]
> I think you can just leave the s3 part out? The question is do we wanna have integration tests for that s3 stuff?

Removed. Integration tests containing S3 would be great. I just did not find the time to re-implement them (see #32).

**Comment by siegfriedweber** [comment]
> I think you can just leave the s3 part out? The question is do we wanna have integration tests for that s3 stuff?

Removed. Integration tests containing S3 would be great. I just did not find the time to re-implement them (see #32).

**Review by sbernauer — APPROVED** [review]
Test LGTM

File: trino-operator/tests/kuttl/smoke/00-assert.yaml [hunk 1]
```diff
@@ -2,14 +2,20 @@
 apiVersion: kuttl.dev/v1beta1
 kind: TestAssert
 metadata:
-  name: install-hive
-timeout: 720
+  name: minio
+timeout: 900
+---
+apiVersion: v1
```
[the remark this answers is not in the document]
Comment (maltesander) [reply trino-operator/tests/kuttl/smoke/00-assert.yaml:8]: Well it should be there to access it later :)

**Review by adwk67 — APPROVED** [review]
Tests LGTM.

**Pull request merged**